docs/rfcs/0002/readme.md

```diff
@@ -37,21 +34,58 @@ The proposal below might take a couple of read-throughs; I've also added a concr
 2. The AUTHOR works with the ADMINISTRATOR to define:
     1. a unique name for the program to be referred by USER
     2. the pod template specification for executing their program
-    3. the NNF storage requirements described above. 
+    3. the NNF storage requirements described above.
```

Comment:
Period or no period, that is the question. Based on the lines above, no period. I'm ambivalent.

Reply:
I struggled with this. I'm a period guy, but that's not consistent with the rest of the document.